config/gke-system-stats-monitor.json

```diff
@@ -0,0 +1,55 @@
+{
+	"net": {
+		"metricsConfigs": {
+			"net/rx_bytes": {
```

Comment:
The indent is weird.

Reply:
I don't know what you mean but I've made this consistent (2 spaces each level)